packages/core/saved-objects/core-saved-objects-migration-server-internal/src/kibana_migrator.ts

```diff
@@ -119,6 +119,10 @@ export class KibanaMigrator implements IKibanaMigrator {
     this.esCapabilities = esCapabilities;
   }
 
+  public getDocumentMigrator() {
+    return this.documentMigrator;
+  }
```

Comment:
We needed access to `documentMigrator.isDowngradeRequired` from higher layers, and I didn't want to expose yet another bridge API (especially given we may want to have more control / access on the doc migrator later if we do a better integration of encryption within the SOR), so I added this API to access the Kibana migrator's underlying doc migrator (and we should have done that a while ago ihmo)